DOCS-2033: App term replaced with Integration for Lightspeed core (#1069)
MultiSafepay/docs: content/integrations/hosted/lightspeed_core/_index.md

logo: "/logo/Integrations/Lightspeed.svg"
weight: 10
title_short: "Lightspeed"

> description_short: "Easily integrate MultiSafepay payment solutions into your Lightspeed webshop with the free integration."

Reviewer: ```suggestion
description_short: "Easily integrate MultiSafepay payment solutions into your Lightspeed webshop with the free core integration."
```

> description: "Easily integrate MultiSafepay payment solutions into your Lightspeed webshop with the free integration.

Reviewer: ```suggestion
description: "Easily integrate MultiSafepay payment solutions into your Lightspeed webshop with the free core integration.
```

> This app is managed by our partner Lightspeed. For support, please contact [Lightspeed](https://www.lightspeedhq.nl/support) directly."

Reviewer: ```suggestion
This core integration is managed by our partner Lightspeed. For support, please contact [Lightspeed](https://www.lightspeedhq.nl/support) directly."
```
